style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: aqua;
+}
```

Comment:
- [ ] Kindly use the class name from your index file to style the content of the `h1`.

Reply:
Hi @Qoosim recommended changes have been made. 

Github workflows fixed - there was a typo in the workflows directory.

Thanks,
E.